File: Python/fizz_buzz.py
def solution(value):
    for i in range(1,value+1):
        if i % 15 == 0:
            print('fizzbuzz')
        elif i % 3 == 0:
            print('fizz')
        elif i % 5 == 0:
            print('buzz')
        else:
            print(i)


if __name__ == '__main__':
    val = int(input('Enter a value to fizzbuzz till: '))
    solution(val)
        

File: JS/fizzBuzz.js
const solution = (number) => {
    for(i = 1; i<=number; i++){
        result = ''
        if(i % 3 === 0){
            result += 'fizz'
        }
        if(i % 5 === 0){
            result += 'buzz'
        }

        console.log(result === '' ? i : result)
    }
}

solution(15)
